plugins/explain.py

```diff
@@ -83,6 +83,16 @@ class Explain(BotPlugin):
                      'newcomer Pull Request\n- merge a difficulty/low Pull '
                      'Request\n- review at least a difficulty/low or higher '
                      'Pull Request'
+        'do not ask to ask': 'This is a lazy way of asking your question '
+                             'because  you\'re asking for more than what '
+                             'you think you\'re asking. The solution is not '
+                             'to ask to ask, but just to ask by actaully '
```

Comment:
You haven't explained what __don't "ask to ask"__ actually means in a proper way (like describe the specific problem, tell us what you did to solve and didn't worked out, what others might have to do to regenerate the problem) :wink: